client/asset/zec/shielded_rpc.go

```diff
@@ -52,7 +67,153 @@ type unifiedReceivers struct {
 	Sapling     string `json:"sapling"`
 }
 
+// favoredAddressAndPolicy returns the best address and the associated privacy
+// policy for a unified address.
+func (r *unifiedReceivers) favoredAddressAndPolicy() (string, privacyPolicy, error) {
+	switch {
+	case r.Orchard != "":
+		return r.Orchard, FullPrivacy, nil
+	case r.Sapling != "":
+		return r.Sapling, AllowRevealedRecipients, nil
+	case r.Transparent != "":
+		return r.Transparent, AllowRevealedAmounts, nil
```

Comment:
Also the context for this method is the *to* address for `z_sendmany`, correct?  
And only when sending *from* a shielded (orchard) address?
So while it feels nice as a method, it seems like a pure helper function for just `(*zecWallet).SendShielded` in zec.go i.e. `func favoredToAddressAndSendPolicy(r *unifiedReceivers)`.

But I think the sapling and transparent policies here are switched.

The docs for `z_sendmany` describe these as

```
- "FullPrivacy": Only allow fully-shielded transactions (involving a single shielded value pool).
- "AllowRevealedAmounts": Allow funds to cross between shielded value pools, revealing the amount
  that crosses pools.
- "AllowRevealedRecipients": Allow transparent recipients. This also implies revealing
  information described under "AllowRevealedAmounts".
- "AllowRevealedSenders": Allow transparent funds to be spent, revealing the sending
  addresses and amounts. This implies revealing information described under "AllowRevealedAmounts".
```

(`AllowRevealedRecipients` and  `AllowRevealedSenders` both imply `AllowRevealedAmounts`.)

If we are sending _to_ a transparent address, I would expect `AllowRevealedRecipients`.  And if we are sending to Sapling, I would expect just `AllowRevealedAmounts`.